src/plotrecipes/histograms.jl

```diff
@@ -0,0 +1,45 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+@userplot MyPlot
+
+@recipe function f(eh::MyPlot; quantiles=[0.25,0.50,0.75])
```

Comment:
For the histogram you want a type recipe, similar to https://github.com/JuliaEarth/GeoStatsBase.jl/blob/e06933f23530778c6a64fb7c65ab62219098fc42/src/plotrecipes/domains/point_set.jl#L27

You don't need to create a userplot 👍🏽

Reply:
I would also rename the argument `eh` to simply `hist`, and would then copy/paste the distplot1d recipe content here assuming that the EmpiricalHistogram was computed already.

Reply:
And then we can make sure that the new distplot1d function simply creates a `EmpiricalHistogram` and uses its plot recipe.